### PR TITLE
Tree Component: Export missing TreeItemType

### PR DESCRIPTION
**Pull request by petdud**
## New Behavior

Exporting `TreeItemType`

## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/d7825cf4863aa3d97ba20df84b08953febf5f7fe'>d7825cf4863aa3d97ba20df84b08953febf5f7fe</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/Avatar-mount_base.html">647</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/Avatar-mount.html">653</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/Button-mount_base.html">318</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/Button-mount.html">319</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/Field-mount_base.html">1116</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/Field-mount.html">1134</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/FluentProvider-mount_base.html">718</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/FluentProvider-mount.html">707</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/FluentProviderWithTheme-mount_base.html">79</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/FluentProviderWithTheme-mount.html">80</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">67</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">67</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">78</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">81</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/InfoButton-mount_base.html">14</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/InfoButton-mount.html">8</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/MakeStyles-mount_base.html">841</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/MakeStyles-mount.html">856</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/Persona-mount_base.html">1767</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/Persona-mount.html">1696</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/SpinButton-mount_base.html">1386</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/29532/perf-test-react-components/SpinButton-mount.html">1396</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: packages/react-components/react-tree/src/index.ts [hunk 1]
```diff
@@ -64,6 +64,7 @@ export type {
   TreeItemProps,
   TreeItemState,
   TreeItemSlots,
+  TreeItemType,
```
Comment (ling1726) [inline packages/react-components/react-tree/src/index.ts:67]: you want to run `yarn workspace @fluentui/react-tree build` to regenerate api markdown

**Review by bsunderhus — APPROVED** [review]

File: packages/react-components/react-tree/src/index.ts (hunk 1 above)
Comment (petdud) [reply packages/react-components/react-tree/src/index.ts:67]: done, thank you 👍

**Pull request merged**